Utils/Dataflow/run/data4es-update-samples

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Utility to update data samples for dataflow stages
+#
+# data4es process overview:
+#
+# 009 --> 091(in) --> 025 --> 016 --> 019 --> 069
+#   \                                         /
+#    \----> 091(out) ----> 093 ----> 095 ----/
+
```

Comment:
@mgolosova 
Pipeline 095 | 069 - looks a little bit strange, because -- according to our discussion -- 069 doesn't provide the data processing from 095. Could it be fixed?

Reply:
Taking into account last changes (including, but not limited to the separation of Stage 16 into two stages, 16 and 17), I think I should mark this PR as `[WIP]`: its current version is not exactly corresponding the dataflow version found in master.